### ISSUE TITLE
Complete run - epoch 100 - model F-M - Only resize the image 224x224

### ISSUE THREAD

**Issue by MaraniMatias** [deeplearningrosario/deeplearningforcomputervision#40]
# Runs
**Image layer:** ResNet50
**Image quantity:** 
- Males: 6780 (4520 training - 1130 validation - 1130 testing)
- Females: 5740 (3828 training - 956 validation - 956 testing)

**Epochs**: 100
**Batch size:** 32
**Loss**: Mean Squared Error
**Additional metrics:** Mean Absolute Error
**Optimizer:** Adam(lr=0.001, amsgrad=True)
**Dropout:** 0.45

## Female && male
**Test loss:** 173.23082294628696
**Test MAE:** 10.055264410327974

![history_loss](https://user-images.githubusercontent.com/7859628/45187498-38aa4c00-b207-11e8-8c03-eb09bd9b3083.png)
![history_mean](https://user-images.githubusercontent.com/7859628/45187500-3942e280-b207-11e8-8b36-b764747fcef6.png)

epoch | loss | lr | mean_absolute_error | val_loss | val_mean_absolute_error
-- | -- | -- | -- | -- | --
0 | 1960.38517678612 | 0.001 | 30.5919663827272 | 862.777863865502 | 21.8163038109316
1 | 430.015364211429 | 0.001 | 16.1219999910144 | 1632.55867942159 | 33.2492866552688
2 | 665.440571030533 | 0.001 | 19.6989918344605 | 459.917922754187 | 16.6432694565873
3 | 449.879494550529 | 0.001 | 16.5639842398497 | 554.563705005445 | 18.418012727827
4 | 354.10222182299 | 0.001 | 14.6058291506984 | 275.982508014741 | 12.5430192910813
5 | 346.103704820824 | 0.001 | 14.4389997947484 | 605.758152845608 | 19.4621072057795
6 | 316.346560468614 | 0.001 | 13.849831611732 | 671.273691208456 | 21.7931161708594
7 | 297.283404084787 | 0.001 | 13.4707682499431 | 521.206946105079 | 18.5110931396484
8 | 407.840399441904 | 0.00080000004 | 14.9337513237749 | 2015.50649067631 | 38.3545434390367
9 | 483.217676079553 | 0.00080000004 | 16.1409248448926 | 427.626555196848 | 15.5780673680804
10 | 392.387427863289 | 0.00080000004 | 14.7483546444829 | 553.761967923246 | 18.1540084813265
11 | 315.92804273853 | 0.00064000004 | 13.3548089028776 | 207.236834400795 | 10.7605433221746
12 | 339.0860924129 | 0.00064000004 | 13.5894576800629 | 1467.33934013681 | 28.2915294058371
13 | 307.229266076666 | 0.00064000004 | 13.1330373414007 | 521.959181583511 | 16.4828995195354
14 | 379.893728390572 | 0.00064000004 | 14.7955933206665 | 291.412882145619 | 12.8615236291492
15 | 298.726610981567 | 0.00051200006 | 13.2845839974294 | 321.608495549525 | 14.1475400668663
16 | 257.629340610038 | 0.00051200006 | 12.4694599728294 | 274.481845932519 | 12.1857817330731
17 | 236.106727055402 | 0.00051200006 | 11.974020177917 | 228.466729358882 | 11.702012756984
18 | 232.702063925596 | 0.00040960003 | 11.8606925332838 | 261.350720208992 | 11.9498950504731
19 | 221.801941868444 | 0.00040960003 | 11.6292839242216 | 413.700625630833 | 16.3980133334605
20 | 209.799853541946 | 0.00040960003 | 11.3182698328871 | 250.412818945266 | 12.4719709028791
21 | 202.90793865882 | 0.00032768003 | 11.0136229949644 | 197.847672840093 | 10.4847858809328
22 | 184.070660162041 | 0.00032768003 | 10.613428900901 | 198.258435345336 | 10.664181792496
23 | 179.206131529797 | 0.00032768003 | 10.4797143218731 | 282.237357003348 | 13.0322275372045
24 | 178.68080988008 | 0.00032768003 | 10.4512943008269 | 253.080798755328 | 12.4711838061149
25 | 172.800443143895 | 0.00026214402 | 10.2173108168064 | 241.864377512799 | 11.9750577829714
26 | 171.48439191481 | 0.00026214402 | 10.181391394304 | 173.647180164077 | 9.81243238513102
27 | 173.860391767618 | 0.00026214402 | 10.324430794709 | 168.166946762246 | 9.63211302514959
28 | 157.164997555962 | 0.00026214402 | 9.79002908844667 | 170.722392725007 | 9.83650606087733
29 | 157.838017359954 | 0.00026214402 | 9.72444449398697 | 168.134062096805 | 9.72495475780015
30 | 149.613588114703 | 0.00026214402 | 9.56178003620976 | 175.875713798252 | 9.89355699167956
31 | 146.194321121048 | 0.00026214402 | 9.42799633795643 | 204.729138426447 | 10.9195153162326
32 | 150.866166985315 | 0.00026214402 | 9.44540100321566 | 176.603931854921 | 9.97215348076386
33 | 140.803650176519 | 0.00020971522 | 9.18808482443824 | 159.971689738112 | 9.42231841101057
34 | 134.197534116079 | 0.00020971522 | 9.06436813763014 | 188.618329831761 | 10.0792509172244
35 | 133.980361708169 | 0.00020971522 | 8.98766022934889 | 165.812819703085 | 9.65529991270597
36 | 316.650480086688 | 0.00020971522 | 13.4999785599208 | 346.801831282911 | 14.0397898737102
37 | 239.6760551332 | 0.00016777217 | 12.0039128731699 | 235.183077467482 | 11.2809836578735
38 | 227.699622358497 | 0.00016777217 | 11.667009801001 | 280.556263061406 | 12.207395830639
39 | 200.670099416531 | 0.00016777217 | 11.0381229114578 | 228.217332658923 | 10.9911591199763
40 | 185.697380011083 | 0.00013421774 | 10.6123443899625 | 199.136578099954 | 10.3382867052365
41 | 178.181361660411 | 0.00013421774 | 10.4035073601577 | 199.698331223924 | 10.3117010531695
42 | 167.22596948365 | 0.00013421774 | 10.0806271987608 | 220.850763483678 | 10.383880329955
43 | 163.731153158234 | 0.000107374195 | 9.87443682629924 | 191.832430685012 | 10.0412183041083
44 | 156.656282937635 | 0.000107374195 | 9.71847096595344 | 186.473305736041 | 10.0291420643076
45 | 152.984296373734 | 0.000107374195 | 9.62645257621732 | 180.504448836739 | 9.9226710757496
46 | 151.627111795934 | 0.0001 | 9.52911341355986 | 184.230127365683 | 9.935064164927
47 | 147.963478179771 | 0.0001 | 9.32397996402552 | 182.838067995622 | 10.0770423757493
48 | 238.893641166157 | 0.0001 | 11.2856257036078 | 324.561417225795 | 14.0446975951227
49 | 192.606207518127 | 0.0001 | 10.7021029764348 | 221.580617551721 | 10.9105054036083
50 | 170.476844535356 | 0.0001 | 10.030090188089 | 195.398867598887 | 10.2329523163354
51 | 158.398328670544 | 0.0001 | 9.7180040123706 | 206.722200856341 | 10.5635052659939
52 | 146.149684487431 | 0.0001 | 9.41201594644719 | 186.997735203627 | 10.0388549994874
53 | 146.186525612627 | 0.0001 | 9.36985512538338 | 191.681504700351 | 10.1420284730247
54 | 141.778322103781 | 0.0001 | 9.18575154164218 | 185.4324642997 | 10.0287354141045
55 | 136.704228648399 | 0.0001 | 9.00501607100702 | 185.427804312336 | 10.0413022749947
56 | 140.007339748059 | 0.0001 | 9.1120633813943 | 178.529086375397 | 9.85075187957413
57 | 126.15399073412 | 0.0001 | 8.62370369705105 | 186.59884296099 | 9.99096380052722
58 | 128.770937617567 | 0.0001 | 8.67715391863689 | 183.066724870486 | 10.0140027661566
59 | 120.483334653276 | 0.0001 | 8.47558801209578 | 180.499116321538 | 9.84000402854707
60 | 120.931919900321 | 0.0001 | 8.45836074191186 | 181.886185072755 | 10.0128712722691
61 | 120.44982383739 | 0.0001 | 8.45572615286695 | 185.161648187976 | 9.97265807871393
62 | 118.47972322984 | 0.0001 | 8.39504820979493 | 185.358251855106 | 10.0609493731081
63 | 114.269088046883 | 0.0001 | 8.23756745310565 | 181.550875048395 | 9.88590663917227
64 | 110.697840417808 | 0.0001 | 8.16213225907096 | 182.600747837202 | 9.87660403631106
65 | 110.722011410796 | 0.0001 | 8.04870874800634 | 182.058570752094 | 9.85881177988171
66 | 107.862066436168 | 0.0001 | 7.95565520968801 | 181.844029091006 | 9.90065073304071
67 | 104.24999955218 | 0.0001 | 7.81546235507221 | 185.828717496 | 9.99388019967742
68 | 102.011754656044 | 0.0001 | 7.72019941588595 | 182.539687422755 | 9.93023335922256
69 | 102.712008488595 | 0.0001 | 7.75126862034709 | 200.802738961415 | 10.4312002697704
70 | 100.053146684004 | 0.0001 | 7.62714115694829 | 186.073560544651 | 9.94477490839314
71 | 97.7233602380318 | 0.0001 | 7.55106800757473 | 192.3903628502 | 10.2677232160779
72 | 96.5066285144786 | 0.0001 | 7.47561768315658 | 207.528131004034 | 10.7167649209671
73 | 95.3726109064648 | 0.0001 | 7.49365114560239 | 206.589925069251 | 10.4752862844348
74 | 95.8292708714498 | 0.0001 | 7.42904629202396 | 187.090801015911 | 10.1292701677302
75 | 91.082947281827 | 0.0001 | 7.24290811153023 | 186.785079070957 | 10.0398731844309
76 | 87.1190495730932 | 0.0001 | 7.15979811650575 | 189.105330777191 | 10.0414275803022
77 | 87.1009716569492 | 0.0001 | 7.11162130856663 | 183.415200881739 | 9.88150973333723
78 | 83.3061402580872 | 0.0001 | 6.95209046671364 | 188.376422637032 | 10.0115129346473
79 | 83.5024505235044 | 0.0001 | 6.95908540690065 | 193.401690613846 | 10.1300369986866
80 | 81.2539799740055 | 0.0001 | 6.83704860917563 | 198.852341795471 | 10.4811223917931
81 | 83.5035823352267 | 0.0001 | 6.93245347835941 | 188.802302952124 | 10.0745431903682
82 | 80.547346647927 | 0.0001 | 6.78367872653992 | 201.42560376234 | 10.4191928591047
83 | 79.0386348919944 | 0.0001 | 6.74989855957762 | 186.761939000306 | 10.0677546156447
84 | 77.1651273214709 | 0.0001 | 6.6390649291049 | 184.978947472138 | 10.1319790645391
85 | 80.7083581468393 | 0.0001 | 6.80829740518808 | 202.571221656104 | 10.5030533154196
86 | 75.8073464372139 | 0.0001 | 6.61755130733093 | 190.167048394851 | 10.2939431335873
87 | 77.8333321038078 | 0.0001 | 6.64828304154598 | 185.540551789998 | 10.0752348748972
88 | 80.1983318008986 | 0.0001 | 6.77446797959931 | 203.039369402087 | 10.6248897386077
89 | 75.9060756169727 | 0.0001 | 6.5526654381014 | 180.599098505475 | 9.88781792502618
90 | 72.4146286800368 | 0.0001 | 6.39748622586296 | 193.775180454565 | 10.2592079568572
91 | 73.5247908054383 | 0.0001 | 6.47302400568883 | 187.323607981491 | 10.0766875677575
92 | 72.0572747421082 | 0.0001 | 6.35399102707713 | 185.396389252616 | 10.0082626891388
93 | 71.630097594396 | 0.0001 | 6.3607881833448 | 186.059171462676 | 10.0519130511115
94 | 72.5262124811455 | 0.0001 | 6.4210934346523 | 197.473776088579 | 10.4476162196349
95 | 72.8921326754707 | 0.0001 | 6.41035491607951 | 185.325011161007 | 10.0801401595484
96 | 67.5629523130599 | 0.0001 | 6.15469687625901 | 186.104199088546 | 10.0815655487953
97 | 66.2757615407917 | 0.0001 | 6.12113915623006 | 183.874629209963 | 10.0053372908873
98 | 67.4385609208652 | 0.0001 | 6.18296818900006 | 190.509677195709 | 10.2234024850657
99 | 66.8085125464827 | 0.0001 | 6.10139322829235 | 185.823870204439 | 10.0133464965016
